src/renderer/components/+catalog/catalog.tsx

```diff
@@ -165,14 +165,14 @@ export class Catalog extends React.Component<Props> {
 
   renderIcon(item: CatalogEntityItem<CatalogEntity>) {
     return (
-      <HotbarIcon
-        uid={item.getId()}
+      <EntityIcon
         title={item.getName()}
         source={item.source}
         src={item.entity.spec.icon?.src}
         material={item.entity.spec.icon?.material}
         background={item.entity.spec.icon?.background}
-        onClick={() => this.onDetails(item)}
+        onClick={() => item.onRun(catalogEntityRunContext)}
+        hoverWidth="1.5px"
```

Comment:
This does not feel right.. why do we need this?

Reply:
This fixes https://github.com/lensapp/lens/issues/3133 by the border width be appropriate to the size of the icon.